CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to napari-gui
+
+We welcome your contributions! Please see the provided steps below and never hesitate to contact us.
+
+## Steps to contribute
+
+#### First, make sure you get development version of the repository and create your own branch
+```
+git clone https://github.com/<username>/napari-gui # get the repo
+git checkout dev # checkout to dev branch
+git branch <branchname> # create your own branch
+```
+
+#### Checkout to your own branch for development
+- Ensure that your contribution is in its own branch in your fork of the repository (no other changes should be in the branch). You can it with:
+```
+git checkout BRANCHNAME
+git status
+```
```

Comment:
Maybe include a section about updating the dev branch via `git pull upstream dev` and the current working branch via `git fetch upstream dev; git rebase -Xtheirs upstream/dev` (will need to verify I'm not sure if this is right)